package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "resource_library",
+  "version": "1.0.0",
+  "description": "The backend for resource library",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hjkmines/resource_library.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hjkmines/resource_library/issues"
+  },
+  "homepage": "https://github.com/hjkmines/resource_library#readme",
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "mongoose": "^6.9.0"
+  }
```

Comment:
Hey @kristenking! I was curious if dotenv, express, and mongoose was installed in the client's package.json instead of server's. Will you please confirm if it is installed in the client's or server's package.json?

Reply:
Hello @vmaineng ! The client folder contains its own front-end components and assets, along with a separate package.json file that manages its dependencies.

Everything outside the client folder, such as the server folder, and the highlighted packages above belong to the back-end and is managed by a separate package-lock.json and package.json files. This separation allows us to manage the dependencies for each part of the application independently and ensures that the front-end and back-end can evolve independently of each other. 

Please let me know if you have any questions or if there is anything else I can help with.